Sources/NIO/Bootstrap.swift

```diff
@@ -45,6 +45,10 @@
 ///     try! channel.closeFuture.wait() // wait forever as we never close the Channel
 /// ```
 ///
+/// The returned `ServerSocketChannel` by `bind` is the server channel and will fire the accepted `SocketChannel`s through the `ChannelPipeline` via `fireChannelRead`
+/// and so operate on `Channel` as inbound messages. Outbound messages are not supported by `ServerSocketChannel`, which means each write attempt will be failed.
+///
+/// The accepted `SocketChannel` will operate on `ByteBuffer` as inbound and `IOData` as outbound messages.
```

Comment:
Let's change this to: "The `EventLoopFuture` returned by `bind` will fire with a `ServerSocketChannel`. This is the channel that owns the listening socket. Each time it accepts a new connection it will fire a `SocketChannel` through the `ChannelPipeline` via `fireChannelRead`: as a result, the `ServerSocketChannel` operates on `SocketChannel`s as inbound messages. Outbound messages are not supported on a `ServerSocketChannel` which means that each write attempt will fail.

Accepted `SocketChannel`s operate on `ByteBuffer` as inbound data, and `IOData` as outbound data."

Reply:
@Lukasa I think we should still say: as a result, the `ServerSocketChannel` operates on `Channel`s as inbound messages...

`SocketChannel` is not public so it may be confusing otherwise if the user tries to use it as `InboundIn` and can't. WDYT ?

Reply:
Sure, that's fine.